system/services/admin/DataManagerService.cfc

```diff
@@ -503,7 +503,8 @@ component {
 			var fieldRelationship = _getPresideObjectService().getObjectProperties( arguments.objectName )["#orderByField#"].relationship ?: "";
 
 			if ( fieldRelationship == "many-to-one" ) {
-				var relatedLabelField = _getFullFieldName( "label", _getPresideObjectService().getObjectProperties( arguments.objectName )["#orderByField#"].relatedTo );
+				var relatedToObjectField = _getPresideObjectService().getObjectProperties( arguments.objectName )["#orderByField#"].relatedTo;
+				var relatedLabelField    = _getFullFieldName( _getPresideObjectService().getObjectAttribute( relatedToObjectField, "labelfield", "label" ), relatedToObjectField );
```

Comment:
I _think_ we should just be able to change `"label"` to `"${labelfield}"`, rather than requiring an extra line of code here. Are you able to try that @nelsonchuah?